### PR TITLE
Aftershock: Make moxphore type enemies neutral to zombies

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
Mods "Aftershock: Make moxphore type enemies neutral to zombies"

#### Purpose of change

Although making these two factions hostile with each other was interesting, the moxphore foes spawning from formless ruins and other potential surface locations caused a considerable balance issues. Since they allowed beginner characters to clear entire tows through simple kiting the moxphore as an anti-zombie bodyguard

#### Describe the solution
Zombies an moxphores are neutral to each other

#### Testing

Spawn relevant foes and notice no infighting.

### CODE REVIEW

**Comment by actual-nh** [comment]
You'll need to change the zombie faction to likewise be neutral to moxies - otherwise, the second the zombies get riled up, they'll attack the moxies without the moxies being able to counterattack.

File: data/mods/Aftershock/mobs/monster_faction.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "type": "MONSTER_FACTION",
     "name": "moxie",
     "base_faction": "mutant",
-    "neutral": "bio_machine",
+    "neutral": [ "bio_machine", "zombie" ],
     "by_mood": [ "small_animal", "herbivore" ],
     "hate": [ "insect", "human", "mutant", "WraitheonRobotics", "bot" ]
   },
```
Comment (actual-nh) [inline data/mods/Aftershock/mobs/monster_faction.json:43]:
```suggestion
  },
  {
  "type": "MONSTER_FACTION",
    "name": "zombie",
    "copy-from": "zombie",
    "neutral": [ "moxie" ]
  },
```
Probably needs some reformatting, and preferably would be located later in the file.

**Review by Mom-Bun — APPROVED** [review]
Just want to say, this was some (hopefully) useful data on faction in-fighting maybe?

So we will hopefully have some info on how to go about it in AFS

File: data/mods/Aftershock/mobs/monster_faction.json (hunk 1 above)
Comment (John-Candlebury) [reply data/mods/Aftershock/mobs/monster_faction.json:43]: Thank you! I forget relationships aren't forcibly symmetric.

**Pull request merged**